specification/azuredependencymap/DependencyMap.Management/main.tsp

```diff
@@ -233,6 +246,27 @@ union SourceType {
   string,
 }
 
+@doc("Model representing the result of the export dependencies asynchronous operation.")
+model ExportDependenciesOperationResult {
+  @doc("The overall arm status of the operation. It has one of the terminal states - Succeeded/Failed/Canceled.")
+  status: string;
+
+  @doc("Contains error details if status is Failed/Canceled.")
+  error?: Azure.ResourceManager.Foundations.ErrorDetail;
+
+  @doc("Properties for export dependencies. These should only be set if the status is Succeeded.")
+  properties?: ExportDependenciesResultProperties;
+}
+
+@doc("Model representing properties returned upon successful completion of the export dependencies asynchronous operation.")
+model ExportDependenciesResultProperties {
+  @doc("The SAS URI of the blob containing the exported dependencies data.")
+  exportedDataSasUri?: string;
+
+  @doc("A status message returned by the service with additional context about the export dependencies operation.")
+  statusMessage?: string;
```

Comment:
>statusMessage [](http://example.com/codeflow?start=2&length=13)

what are diff values of the message? Should this be within properties or it is extension to status field?